infrastructure/tari_script/Cargo.toml

```diff
@@ -21,6 +21,4 @@ serde = "1.0.136"
 sha2 = "0.9"
 sha3 = "0.9"
 thiserror = "1.0.30"
-
-[dev-dependencies]
```

Comment:
Why has this been removed?

Reply:
We are now in need of the `rand` library on `[dependencies]`', but can add it to `[dev-dependencies]` as well.

Reply:
cool, github does not show that only the rand lib is left.

Reply:
I don't see any code changes that require rand as a dependency, I think the dev-dependency can be re-added?